### ISSUE TITLE
Compilation of sigTest.cpp fails on GCC 5.4

### ISSUE THREAD

**Issue by mgaudet** [eclipse/omr#328]
```
cc  -I. -I../util -I../../third_party/gtest-1.7.0 -I../../third_party/gtest-1.7.0/include -I../../fvtest/omrGtestGlue  -DUT_DIRECT_TRACE_REGISTRATION -I../../include_core -I../../nls -DLINUX -D_REENTRANT -D_FILE_OFFSET_BITS=64 -DS390 -D_LONG_LONG -DJ9VM_TIERED_CODE_CACHE -DS39064 -c   -fno-exceptions -fno-rtti -fno-threadsafe-statics -fPIC -g  -fno-strict-aliasing -Wreturn-type -Werror -Wall -Wno-non-virtual-dtor -O3 -march=z900  -mtune=z9-109 -mzarch -fexceptions  -o sigTest.o sigTest.cpp
sigTest.cpp: In function 'omr_error_t test(TestAction)':
sigTest.cpp:330:28: error: format '%ld' expects argument of type 'long int', but argument 8 has type 'int' [-Werror=format=]
        onStack ? "" : "no ");
```

```
$ cc --version
cc (Ubuntu 5.4.0-6ubuntu1~16.04.2) 5.4.0 20160609
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

(this is the default LinuxOne Ubuntu 16.04 GCC )

**Comment by mgaudet** [comment]
May have to do with configuration?

 ` make -f run_configure.mk OMRGLUE=./example/glue SPEC=linux_390-64`

```C++
                        rc = OMR_ERROR_INTERNAL;
#if (defined(S390) && defined(OMR_ENV_DATA64))
                        printf("Test %d:%d failed at %s:%d. Conditions: %sexisting primary, %sexisting secondary, flags = %ld, signum is %d, handler is 0x%p, %salt signal stack.\n",
#else /* (defined(S390) && defined(OMR_ENV_DATA64)) */
                        printf("Test %d:%d failed at %s:%d. Conditions: %sexisting primary, %sexisting secondary, flags = %d, signum is %d, handler is 0x%p, %salt signal stack.\n",
#endif /* (defined(S390) && defined(OMR_ENV_DATA64)) */
```

**Comment by 0xdaryl** [comment]
This was fixed in #1560.
